rpc/context.go

```diff
@@ -53,7 +53,7 @@ func NewServer(ctx *Context) *grpc.Server {
 // Context contains the fields required by the rpc framework.
 type Context struct {
 	// Embed the base context.
-	base.Context
+	*base.Context
```

Comment:
it wasn't complaining about anything here, but it complained about the dereference in the constructor copying a mutex.